aip/aog/3001.md

```diff
@@ -0,0 +1,127 @@
+---
+aip:
+  id: 3001
+  state: draft
+  created: 2019-10-03
+  updated: 2019-10-03
+  type: process
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /3001
+---
+
+# Actions on Google AIP Process
+
+This AIP extends [AIP-1][] with details specific to Actions on Google AIPs
+("AoG AIPs"). Any details of [AIP-1][] not modified or contradicted by this AIP
+also apply to AoG AIPs.
+
+## Stakeholders
+
+As with any process there are many different stakeholders when it comes to
+reviewing and working with AIPs. Below is a summary of the escalation path
+starting with the API producer.
+
+```graphviz
+digraph d_front_back {
+  rankdir=BT;
+  ranksep=0.3;
+  node [ style="filled,solid" shape=box fontname="Roboto" ];
+
+  producer [ label="API Producer" ];
+  editors [ label="AIP Editors" ];
+  aog_editors [ label="AoG AIP Editors" ];
+  tl_infra [ label="Infrastructure TL" ];
+  tl_design [ label="Design TL" ];
+  tl [ label="TL" ];
+
+  producer -> aog_editors;
+  aog_editors -> editors;
+  editors -> tl_infra -> tl;
+  editors -> tl_design -> tl;
+}
+```
+
+### Actions on Google Editors
+
+The Actions on Google editors are the set of people who make decisions on AoG
+AIPs before escalation to the general editors defined in [AIP-1][].
+
+The list of AoG AIP editors is currently:
+
+- Ali Ibrahim ([@ahahibrahim][])
+- Richard Frankel ([@rofrankel][])
+- Silvano Luciani ([@silvolu][])
+
+The AoG editors have the same responsibilities as the general editors. They
+also have the additional responsibility of establishing correctness of, and
+leadership support for, the contents of AoG AIPs.
+
+AoG AIP editorship is by invitation of the current AoG editors.
+
+## States
+
+AoG AIPs use the states defined in [AIP-1][], except that the "Reviewing" state
+is divided into two states, "AoG Reviewing" and "Reviewing":
```

Comment:
Based on [Luke's reply on the other PR](https://github.com/googleapis/aip/pull/326#issuecomment-540893947), we should fix this and have only one "Reviewing" state.